crates/turbo-tasks-fs/src/lib.rs

```diff
@@ -1242,11 +1242,22 @@ impl File {
         self.meta.content_type.as_ref()
     }
 
+    /// Returns the status code associated with this file.
+    pub fn status_code(&self) -> Option<u16> {
+        self.meta.status_code
+    }
+
     pub fn with_content_type(mut self, content_type: Mime) -> Self {
         self.meta.content_type = Some(content_type);
         self
     }
 
+    /// Sets the status code associated with this file.
+    pub fn with_status_code(mut self, status_code: u16) -> Self {
```

Comment:
I think this is the wrong level for this API, and I'm glad the follow up PR is removing it.